packages/bot-skeleton/src/scratch/dbot.js

```diff
@@ -295,14 +295,19 @@ class DBot {
      * Checks whether the workspace contains all required blocks before running the strategy.
      */
     checkForRequiredBlocks() {
-        if (!hasAllRequiredBlocks(this.workspace)) {
+        let message;
+
+        if (!hasAllRequiredBlocks(this.workspace)) message = 'missing from';
+        if (!isAllRequiredBlocksEnabled(this.workspace)) message = 'disabled in';
```

Comment:
Both should be localised messages. I recommend to localise the full message so as to provide proper context to the translators.

Reply:
@aaron-binary Actually at first I did this, but I changed it because of some comments, please take a look again.